packages/record-tuple-polyfill/src/tuple.test.js

```diff
@@ -110,29 +110,26 @@ test("Tuple.of", () => {
 describe("all and only the specified prototype methods exist", () => {
     const list = ([str]) => str.trim().split(/\s+/g);
 
-    // MISSING: Symbol.toStringTag
     const names = list`
-        constructor valueOf length
+        constructor valueOf length with
         popped pushed reversed shifted sliced
         sorted spliced concat includes indexOf join
         lastIndexOf entries every filter find findIndex
         forEach keys map reduce reduceRight some
         unshifted toLocaleString toString values
-    `.concat(Symbol.iterator);
+    `.concat(Symbol.iterator, Symbol.toStringTag);
 
     test.each(names)(".%s", name => {
         // We can't use expect().toHaveProperty because its doesn't support symbols
         expect(hasOwn(Tuple.prototype, name)).toBe(true);
 
-        if (name !== "length") {
+        if (name !== "length" && name !== Symbol.toStringTag) {
             expect(Tuple.prototype[name]).toEqual(expect.any(Function));
         }
     });
 
     test("no extra properties", () => {
-        expect(Object.getOwnPropertyNames(Tuple.prototype)).toHaveLength(
-            names.length,
-        );
+        expect(Reflect.ownKeys(Tuple.prototype)).toHaveLength(names.length);
```

Comment:
`Object.getOwnPropertyNames` doesn't return symbols, so it didn't get the correct number of properties and didn't catch the missing `with` in the expected properties list.